src/Components/Milestones.js

```diff
@@ -26,7 +26,8 @@ function Milestones({ milestones }) {
     <Card
       title={milestone.title}
       subTitle={milestone.date}
-      className="my-5 md:mx-5 shadow-8"
+      className="my-5 md:mx-5 shadow-8  "
+      style={{ borderRadius: '20px' }}
```

Comment:
Another PR to add the border radius is already underReview! So Revert the changes from `Milestone.js`
```suggestion
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable